api/routes.py

```
@@ -248,23 +256,21 @@ async def correct_food_text_analysis(
 
 
-@router.post(
-    "/exercise/correct",
-    response_model=ExerciseAnalysisResult,
-    summary="Correct exercise analysis",
-    tags=["Exercise"],
-)
+@router.post("/exercise/correct")
 async def correct_exercise_analysis(
     request: ExerciseCorrectionRequest,
     gemini: GeminiService = Depends(get_gemini_service),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)